triage/interactive.js

```diff
@@ -50,9 +50,12 @@ function readOptions() {
     date: read('date'),
     ci: read('job-ci'),
     pr: read('job-pr'),
-    reText: read('filter-text'),
-    reJob: read('filter-job'),
-    reTest: read('filter-test'),
+    reText: read('filter-include-text'),
+    reJob: read('filter-include-job'),
+    reTest: read('filter-include-test'),
+    reXText: read('filter-exclude-text'),
+    reXJob: read('filter-exclude-job'),
+    reXTest: read('filter-exclude-test'),
```

Comment:
these aren't part of the url params are they? if so, would be good to keep those stable